pawn/tf2items.inc

```diff
@@ -34,92 +34,92 @@
  * so please make sure you fill out everything: Classname, item definition index,
  * quality, level and attributes.
  * 
- * @param iClient	Client that the item will be generated for.
+ * @param client	Client that the item will be generated for.
  * @param hItem		Handle to the TF2Items object that we'll be operating with.
  * @return			Entity index of the newly created item.
  */
-native TF2Items_GiveNamedItem(iClient, Handle:hItem);
+native int TF2Items_GiveNamedItem(int client, Handle hItem);
 
 /**
  * Creates a TF2Items object, wich can be used to override the parameters of an
  * item before it's given to the client, or to create a new completely new one
  * with GiveNamedItem. Remember to free the object with CloseHandle()
  *
- * @param iFlags		Flags used to specify what to override.
+ * @param flags		Flags used to specify what to override.
  * @return				Handle to the TFItems object.
  */
-native Handle:TF2Items_CreateItem(iFlags);
+native Handle TF2Items_CreateItem(int flags);
 
 /**
  * Sets the flags to determine what the item will override on the GiveNamedItem.
  * Use the OVERRIDE_ defines to set what you will be changing.
  *
  * @param hItem			Handle to the TF2Items object that we'll be operating with.
- * @param iFlags		Flags used to specify what to override.
+ * @param flags		Flags used to specify what to override.
  * @noreturn
  */
-native TF2Items_SetFlags(Handle:hItem, iFlags);
+native void TF2Items_SetFlags(Handle hItem, int flags);
 
 /**
  * Sets the new entity classname used for the item's entity.
  *
  * @param hItem				Handle to the TF2Items object that we'll be operating with.
- * @param strClassName		New classname to use for the entity.
+ * @param classname		New classname to use for the entity.
  * @noreturn
  */
-native TF2Items_SetClassname(Handle:hItemOverride, String:strClassName[]);
+native void TF2Items_SetClassname(Handle hItem, char[] classname);
 
 /**
  * Sets the item's Definition Index, wich tells the game what item it is. Each
  * weapon/hat/item has an unique definition index. Find these out in the
  * game_items.txt
  * 
  * @param hItem				Handle to the TF2Items object that we'll be operating with.
- * @param iItemDefIndex		New definition index.
+ * @param weapon		New definition index.
  * @noreturn
  */
-native TF2Items_SetItemIndex(Handle:hItem, iItemDefIndex);
+native void TF2Items_SetItemIndex(Handle hItem, int weapon);

```

Comment:
Not sure `weapon` makes a good name for item definition index (here and elsewhere), `weapon` implies an entity index to me.